angular-app/src/app/left-menu/left-menu.component.ts

```diff
@@ -9,12 +13,87 @@ import { AuthService } from '../auth.service';
 })
 export class LeftMenuComponent implements OnInit {
 
-  constructor(private authService: AuthService) { }
+  constructor(
+    private authService: AuthService,
+    public dialog: MatDialog,
+  ) { }
 
   ngOnInit(): void {
   }
 
   logout() {
     this.authService.logout();
   }
+
+  openCreateProjectDialog(): void {
+    const dialogRef = this.dialog.open(CreateProjectDialog);
+  }
+}
+
+@Component({
+  selector: 'create-project-dialog',
+  templateUrl: 'create-project-dialog.html',
+})
+export class CreateProjectDialog implements OnInit {
```

Comment:
This component should probably be declared in another file for organization purposes. create-dialog.component.ts

Reply:
Done